sympy/utilities/lambdify.py

```diff
@@ -50,9 +50,9 @@
     "E": "e",
     "I": "j",
     "ln": "log",
-    #"lowergamma":"lower_gamma",
+    "lowergamma":"gammainc",
```

Comment:
yes, they are used to translate the keyword used by sympy to the ones used by mpmath.